backend/trojan_dining/webscraper/webscraper.py

```diff
@@ -101,27 +101,54 @@ def make_menu(soup):
             # so have to get a list of foods in each stations here
             menu_items_list_list = hall.find_all("ul")
             list_of_stations = []
-
-            for i in range(len(station_names_html)):  # loop through all the stations
-                station_name = station_names_html[i].text  # station name
-                list_of_menu_items = []
-                # when theres no items program bugs out; my workaround
-                if station_name != "No items to display for this date":
-                    menu_items_html = menu_items_list_list[i].find_all(
-                        "li")  # find all the food items
-                    for menu_item in menu_items_html:  # iterate through all the food
-                        # get the name of the food item
-                        menu_item_name = menu_item.find(
-                            text=True, recursive=False)
-                        # get all the allergens for that food item
-
-                        allergens_html = menu_item.find_all("i")
-                        list_of_allergens = [allergen.find(
-                            "span").text for allergen in allergens_html]
-                        list_of_menu_items.append(
-                            MenuItem(menu_item_name, list_of_allergens))
-                list_of_stations.append(
-                    Station(station_name, list_of_menu_items))
+            # check if dining hall is evk
+            if(dining_hall_name.strip() == "Everybody's Kitchen"):
```

Comment:
Lets not use parentheses in python `if` conditions. The linter will catch this!

Reply:
Sorry. my bad